src/Tizen.Network.Stc/Tizen.Network.Stc/StcEnumerations.cs

```diff
@@ -0,0 +1,253 @@
+    }
+
+    /// <summary>
+    /// Enumeration for Stc time period.
+    /// </summary>
+    public enum TimePeriodType
+    {
+        /// <summary>
+        /// Hour
+        /// </summary>
+        Hour = 3600,
+        /// <summary>
+        /// Day
+        /// </summary>
+        Day = 86400,
+        /// <summary>
+        /// Week
+        /// </summary>
+        Week = 604800,
+        /// <summary>
+        /// Month
+        /// </summary>
+        Month = 2419200
+    }
+
+    /// <summary>
+    /// Enumeration for network interface type.
+    /// </summary>
+    public enum NetworkInterface
+    {
+        /// <summary>
+        /// Network interface type is uncategorized.
+        /// </summary>
+        Others = 0,
+        /// <summary>
+        /// Network interface type is datacall. 
+        /// </summary>
+        Datacall,
+        /// <summary>
+        /// Network interface type is  wifi.
+        /// </summary>
+        Wifi,
+        /// <summary>
+        /// Network interface type is  wired.
+        /// </summary>
+        Wired,
+        /// <summary>
+        /// Network interface type is bluetooth.
+        /// </summary>
+        Bluetooth,
+        /// <summary>
+        /// Type for any network interface.
+        /// </summary>
+        All
+    }
+
+    /// <summary>
+    /// Internal Enumeration for network roaming type
+    /// </summary>
+    internal enum RoamingType
+    {
+        /// <summary>
+        /// Not in roaming.
+        /// </summary>
+        Disabled,
+        /// <summary>
+        /// In roaming.
+        /// </summary>
+        Enabled
+    }
+
+    /// <summary>
+    /// Enumeration for hardware network protocol type.
+    /// </summary>
+    public enum NetworkProtocol
+    {
+        /// <summary>
+        /// Network protocol uncategorized.
+        /// </summary>
+        Others,
+        /// <summary>
+        /// Network no service.
+        /// </summary>
+        NoService,
+        /// <summary>
+        /// Network emergency.
+        /// </summary>
+        Emergency,
+        /// <summary>
+        /// Network search 1900.
+        /// </summary>
+        Search,
+        /// <summary>
+        /// Network 2G.
+        /// </summary>
+        Datacall_2G,
+        /// <summary>
+        /// Network 2.5G.
+        /// </summary>
+        Datacall_2_5G,
+        /// <summary>
+        /// Network EDGE.
+        /// </summary>
+        Datacall_2_5G_Edge,
+        /// <summary>
+        /// Network UMTS.
+        /// </summary>
+        Datacall_3G,
+        /// <summary>
+        /// Network HSDPA.
+        /// </summary>
+        Hsdpa,
+        /// <summary>
+        /// Network LTE.
+        /// </summary>
+        Lte
+    }
+
+    /// <summary>
+    /// Enumeration for monitored application state.
+    /// </summary>
+    public enum ApplicationStateType
+    {
+        /// <summary>
+        /// Uncategorized state.
+        /// </summary>
+        Others,
```

Comment:
It could be Others?

Reply:
Yes, it is possible to have it. I reffered to file
'stc-plugin-monitor-app.c' -> '__app_update_statistics':

`
106         if (strstr(stat.app_id, "_BACKGROUND")) {
107             stat.ground = STC_APP_STATE_BACKGROUND;
108         } else {
109             if (strstr(stat.app_id, "TOTAL_"))
110                 stat.ground = STC_APP_STATE_UNKNOWN;
111             else
112                 stat.ground = STC_APP_STATE_FOREGROUND;
113         }
`